### ISSUE TITLE
Minimum system requirements to run the MOSEI experiment?

### ISSUE THREAD

**Issue by anuragrawal2024** [declare-lab/Multimodal-Infomax#17]
Hi @Clement25 ,

I am trying to train the model with the MOSEI dataset. I have a linux machine with 16GB RAM. OOMD service is killing the python process since the script needs a lot of memory. I think it's happening during the dataset creation where the large MOSEI pickle file is loaded and processed. Is there a minimum system requirements guide to run this experiment? If not, do you have any recommendation to handle this issue? Thanks!

![image](https://github.com/declare-lab/Multimodal-Infomax/assets/140737044/866fb6e0-92a5-4595-927d-317f2f227cf8)

**Comment by Clement25** [comment]
Yes, the minimum RAM size is around 32GB. You can try gradient accumulation with smaller batch sizes.
